src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -6,19 +6,20 @@
   position: absolute;
   background: $primary;
   top: 66px;
-  left: 50%;
+  // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
```

Comment:
@mike-lvov What does this '40%' here refer to? It doesn't seem to align with the code? Is it outdated/can it be deleted?